### PR TITLE
DOC: Updating contributor's guide.  More info on doc strings, PEP8, git branches and more.

### PR DESCRIPTION
**Pull request by zssherman**
@rcjackson Question, so the doc for contributors guide fails due to the python code blocks trying to import the actually library but I figured I save a space for the pull request, but is there a way to display python code in the rst file without using the actually act code?

### CODE REVIEW

**Comment by rcjackson** [comment]
I thought ..code-block :: python did not actually execute the code. What is your error?

**Comment by zssherman** [comment]
@rcjackson:
Warning, treated as error:
[autosummary] failed to import 'TimeSeriesDisplay': no module named TimeSeriesDisplay
Makefile:20: recipe for target 'html' failed
make: *** [html] Error 2

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -54,51 +54,394 @@ If you are proposing a feature:
...
+
+        """
+        act.plotting.TimeSeriesDisplay
+        ==============================
+        Class for creating timeseries plots from ACT datasets.
+
+        .. autosummary::
+            :toctree: generated/
+
+            TimeSeriesDisplay
```
Comment (rcjackson) [inline CONTRIBUTING.rst]: I would use a different doc string here? I think it's trying to import an RST file called TimeSeriesDisplay for some reason.

**Comment by rcjackson** [comment]
I think it's looking for a module called TimeSeriesDisplay in the example docstring you have. This sounds like an issue we may need to raise with Sphinx autosummary. I would try using a different docstring example to see if that fixes the problem.

**Comment by zssherman** [comment]
@rcjackson I just removed the code block because there kinda was already an explanation.

**Comment by rcjackson** [comment]
Yeah I'm not sure why those aren't working. They should be, but I think it's a bug in sphinx in how it interprets doc strings in a code block. I'll commit for now and then we can work on a way to get the docs for __init__.py in. Perhaps we can just provide a hyperlink to an example __init__.py.

**Pull request merged**